contrib/inventory/ec2.py

```diff
@@ -590,7 +590,7 @@ def get_rds_instances_by_region(self, region):
                 error = self.get_auth_error_message()
             if not e.reason == "Forbidden":
                 error = "Looks like AWS RDS is down:\n%s" % e.message
-            self.fail_with_error(error, 'getting RDS instances')
+                self.fail_with_error(error, 'getting RDS instances')
```

Comment:
That would be a separate pull request. Given that fail_with_error already exits, a raise would be pointless.